### PR TITLE
Update article-changer-v3.js

### PR DESCRIPTION
**Pull request by ArnaudCrevier**
add articleChanger.leave and articleChanger.leaving

### CODE REVIEW

File: src/com/article-changer-v3.js [hunk 1]
```diff
@@ -13,8 +13,16 @@
 			ctn.css({
 				minHeight: current.height() + 'px'
 			});
+			App.mediator.notify('articleChanger.leaving', {
+				item: current,
```
Comment (fhamon) [inline src/com/article-changer-v3.js]: we should drop `item` no one uses it and we should all stick to one.

Comment (nitriques) [reply src/com/article-changer-v3.js]: @fhamon so only article ?

Comment (fhamon) [reply src/com/article-changer-v3.js]: Yep. We use `article` everywhere.

Comment (nitriques) [reply src/com/article-changer-v3.js]: @fhamon ok

**Comment by nitriques** [comment]
Merged as bd8560ffaf

**Pull request closed without merging**